tests/Unit/NumericalAlgorithms/Interpolation/Test_InterpolationTargetReceiveVars.cpp

```diff
@@ -71,6 +71,39 @@ struct CompletedTemporalIds;
 
 namespace {
 
+// In the test, we don't care what SendPointsToInterpolator actually does;
+// we care only that SendPointsToInterpolator is called with the
+// correct arguments.
+template <typename InterpolationTargetTag>
+struct MockSendPointsToInterpolator {
+  template <
+      typename ParallelComponent, typename DbTags, typename Metavariables,
+      typename ArrayIndex,
+      Requires<tmpl::list_contains_v<
+          DbTags, intrp::Tags::TemporalIds<
+                      typename Metavariables::temporal_id::type>>> = nullptr>
+  static void apply(
+      db::DataBox<DbTags>& box,
+      Parallel::ConstGlobalCache<Metavariables>& /*cache*/,
+      const ArrayIndex& /*array_index*/,
+      const typename Metavariables::temporal_id::type& temporal_id) noexcept {
+    using temporal_id_type = typename Metavariables::temporal_id::type;
+    Slab slab(0.0, 1.0);
+    CHECK(temporal_id == TimeStepId(true, 0, Time(slab, Rational(14, 15))));
+    // Increment IndicesOfFilledInterpPoints so we can check later
+    // whether this function was called.  This isn't the usual usage
+    // of IndicesOfFilledInterpPoints; this is done only for the test.
+    db::mutate<intrp::Tags::IndicesOfFilledInterpPoints<temporal_id_type>>(
```

Comment:
Maybe I'm not seeing it, but is there a way you can verify that this mock action was called by checking the state of `intrp::Tags::IndicesOfFilledInterpPoints<temporal_id_type>` at the end of the test?

Reply:
Yes, this is already done on line 498 (check that it hasn't been called yet) and again at line 515 (check that it has been called).

Reply:
Great, thanks! :)